Assets/00_Content/Scripts/Rounds/ScalingData.cs

```diff
@@ -22,6 +22,12 @@ public class ScalingData : ScriptableObject {
 		[Min(0)]
 		public float tableHealth = 100f;
 
+		[Min(0)]
+		public int tableRepairCost = 25;
```

Comment:
Is costs something we want to change based on number of players playing? Feels like costs might be something to keep static?

Reply:
Dunno. But I thought that this might be aa convenient place to put it - makes editing it easier.